cpp/dynamic_community_detection_module/dynamic_community_detection_module.cpp

```diff
@@ -0,0 +1,329 @@
+#include <mg_graph.hpp>
+#include <mg_utils.hpp>
+
+#include "algorithm/dynamic_community_detection.hpp"
+
+constexpr char const *kFieldNode = "node";
+constexpr char const *kFieldCommunityId = "community_id";
+
+constexpr char const *kFieldMessage = "message";
+
+constexpr char const *kDirected = "directed";
+constexpr char const *kWeighted = "weighted";
+constexpr char const *kSimilarityThreshold = "similarity_threshold";
+constexpr char const *kExponent = "exponent";
+constexpr char const *kMinValue = "min_value";
+constexpr char const *kWeightProperty = "weight_property";
+constexpr char const *kWSelfloop = "w_selfloop";
+constexpr char const *kMaxIterations = "max_iterations";
+constexpr char const *kMaxUpdates = "max_updates";
+
+constexpr char const *kCreatedVertices = "createdVertices";
+constexpr char const *kCreatedEdges = "createdEdges";
+constexpr char const *kUpdatedVertices = "updatedVertices";
+constexpr char const *kUpdatedEdges = "updatedEdges";
+constexpr char const *kDeletedVertices = "deletedVertices";
+constexpr char const *kDeletedEdges = "deletedEdges";
```

Comment:
Prefer `std::string_view`.

Reply:
Whereas I understand we prefer `std::string_view` with constant strings, I don't know if that is feasible here. For one, `mgp::proc_add_opt_arg` requires a `const char *` argument, and no suitable conversion function from `const std::string_view` to `const char *` exists.

Reply:
And of course, thanks for all the suggestions!

Reply:
That works :+1: